src/adapters/mc/nrp/Common.h

```diff
@@ -1,54 +1,54 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Is there any change to this file? Some of them show up but it looks like what was added/deleted is exactly the same.

Reply:
Got it, it was moved. I see.